src/tools/post_processor/post_processor/dwarf_string_patcher.cc

```diff
@@ -210,6 +223,11 @@ std::unique_ptr<uint8_t[]> DWARFStringPatcher::RewriteStringSection(
     bool *data_was_modified) {
   assert(data && relocation_table && new_data_length && data_was_modified);
 
+  // TODO(bkase): Actually use the full prefix map
```

Comment:
This won't break anything because the bazel sandbox execution root path will "always" be longer than the path to the local workspace (like ~/code/ios)

"always" is in quotes

In the case that your path to your local workspace is longer than the bazel execution root (virtually never, but I guess you could imagine a universe where that would exist), we will only take the first replacement (aka same behavior as before I touched this diff)